cdap-master/src/test/java/io/cdap/cdap/master/environment/k8s/AppFabricServiceMainTest.java

```diff
@@ -52,36 +44,30 @@ public void testAppFabricService() throws Exception {
 
     Assert.assertEquals(HttpURLConnection.HTTP_OK, response.getResponseCode());
 
-    // Deploy an app
-    LocationFactory locationFactory = new LocalLocationFactory(TEMP_FOLDER.newFolder());
-    Location deploymentJar = AppJarHelper
-        .createDeploymentJar(locationFactory, AllProgramsApp.class);
-
-    URI baseUri = getRouterBaseUri().resolve("/v3/namespaces/default/");
-    url = baseUri.resolve("apps").toURL();
+    // Create a namespace.
+    final String name = "test_namespace";
+    final String description = "test namespace description";
+    URI baseUri = getRouterBaseUri().resolve("/v3/namespaces/" + name);
+    url = baseUri.toURL();
+    String requestBody =
+        String.format("{\"name\":\"%s\", \"description\":\"%s\"}", name, description);
```

Comment:
why this test is changed?

Reply:
AppfabricServiceMain and AppfabricProcessorMain have their own level DB storage.

During App deployment, the entries for the new app are created in the appfabric storage. When schedule is being created in the appfabric processor, it checks for the existing of app and fails as its level DB storage doesn't have the app entries.

Changed this deploy app test to create namespace test.